app/components/DosesDescriptionList/DosesDescriptionList.tsx

```diff
@@ -16,6 +18,14 @@ type DosesDescriptionProps = {
 
 type DosesDescriptionListProps = { dhbData: DhbVaccineDoseDataForIndexPage };
 
+const DisplayPercentage = ({ percentage }: DisplayPercentageProps) => {
```

Comment:
Imo its fine to inline these especially for these smaller components e.g. 

const DisplayPercentage = ({ percentage }: { percentage: number } )